proto/Networking/Requests/Social/CreateCombatChallengeMessage.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package POGOProtos.Networking.Requests.Messages;
```

Comment:
No

Reply:
moved to social

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/6c7ac65e79e4bdec9a6ee0a460b4c4be595c73b0#diff-f75bb0b5c2f39c52efe070172b99e42fR4